resources/tax_type/ca_ab_gst.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Canada - Alberta",
+    "generic_label": "GST",
+    "display_inclusive": false,
+    "zone": "ca_ab_vat",
+    "tag": "CA",
+    "rates": [
+        {
+            "id": "ca_ab_gst_standard",
+            "name": "Standard",
+            "default": true,
+            "compound": true,
```

Comment:
Shouldn't this be FALSE? as Alberta doesn't have PST or is it to go with other taxes? If it should be TRUE then I think we need to do this to the ca_bc_gst too?